packages/integrations/vercel/README.md

```diff
@@ -94,6 +94,8 @@ To configure this adapter, pass an object to the `vercel()` function call in `as
 
 Use this property to force files to be bundled with your function. This is helpful when you notice missing files.
 
+__`astro.config.mjs`__
```

Comment:
Put the name of the file inside the codeblock as a commnet. Like this:

```
// astro.config.mjs
```

That way, it will be shown nicely in the docs